### ISSUE TITLE
data.gov.au

### ISSUE THREAD

**Issue by arminhaller** [AGLDWG/TR#7]
Do we distinguish between http://data.gov.au and state data portals that use a similar pattern, e.g. https://www.data.act.gov.au/

**Comment by dr-shorthair** [comment]
Yes we do distinguish. We have no formal influence over the state portals, though we could hope that they would pay some attention, but typically will be answerable to many masters before they get down to us.
Simon J D Cox
Research Scientist
Land and Waterhttp://www.csiro.au/Organisation-Structure/Flagships/Land-and-Water
CSIRO
E simon.cox@csiro.au T +61 3 9545 2365 M +61 403 302 672
   Physical: Reception Central, Bayview Avenue, Clayton, Vic 3168
   Deliveries: Gate 3, Normanby Road, Clayton, Vic 3168
   Postal: Private Bag 10, Clayton South, Vic 3169
people.csiro.au/C/S/Simon-Coxhttp://people.csiro.au/C/S/Simon-Cox
orcid.org/0000-0002-3884-3420http://orcid.org/0000-0002-3884-3420
researchgate.net/profile/Simon_Cox3https://www.researchgate.net/profile/Simon_Cox3

**Comment by arminhaller** [comment]
Yes, I agree, we may want to then explicitly state: http://data.gov.au.
